Holyhead-Harpies/api-sprint-1-ml
# Product type

## Status
**READY**

## Migrations
NO

## Description
Verbs work on Product Types and validates that all fields are present

## Todos


## Steps to Test or Reproduce
```
git fetch origin product_type
git checkout product_type
bundle
rails db:migrate
rails db:seed
```

1. GET, POST, PUT without product_type_name for ProductTypes table
* Should fail

2. GET, POST, PUT with product_type_name for ProductTypes table
* Should pass

3. DELETE Product 
* Should pass

## Impacted Areas in Application
List general components of the application that this PR will affect:

* ProductTypes Table
